elasticsearch/src/main/scala/com/velocidi/apso/elasticsearch/ElasticsearchBulkInserter.scala

```diff
@@ -21,26 +20,40 @@ import com.velocidi.apso.Logging
  * This actor buffers requests until either the configured flush timer is
  * triggered or the buffer hits the max size.
  */
-class ElasticsearchBulkInserter(bulkInserterConfig: config.Elasticsearch.BulkInserter, client: ElasticClient)
-  extends Actor with ActorLogging {
+class ElasticsearchBulkInserter(esConfig: config.Elasticsearch)
+  extends Actor with Logging {
   import ElasticsearchBulkInserter._
 
   implicit private[this] val ec: ExecutionContext = context.system.dispatcher
 
-  private[this] var esStateListeners = List.empty[ActorRef]
+  private[this] val bulkInserterConfig = esConfig.bulkInserter.getOrElse {
+    val fallback = config.Elasticsearch.BulkInserter(1.second, 10.seconds, 1000, 3)
+    log.warn("Bulk inserter settings for sending documents to Elasticsearch were not found in the config. " +
+      s"A default configuration will be used: $fallback")
+    fallback
+  }
 
   private[this] val maxBufferSize = bulkInserterConfig.maxBufferSize
   private[this] val maxTryCount = bulkInserterConfig.maxTryCount
   private[this] val flushFrequency = bulkInserterConfig.flushFrequency
   private[this] val esDownCheckFrequency = bulkInserterConfig.esDownCheckFrequency
 
+  private[this] var client: ElasticClient = null
+  private[this] var esStateListeners = List.empty[ActorRef]
```

Comment:
Can we remove this? I think this is never used.

Reply:
I think we can. I've removed the state listening logic in cd3182a.